src/main/java/org/obsidiantoaster/generator/rest/ObsidianResource.java

```diff
@@ -264,10 +311,12 @@ public Response downloadZip(JsonObject content,
             Result result = controller.execute();
             if (result instanceof Failed)
             {
-               return Response.status(Status.INTERNAL_SERVER_ERROR).entity(result.getMessage()).build();
+               return Response.status(Status.INTERNAL_SERVER_ERROR).entity(getMessage(result)).build();
```

Comment:
when writing these kinda of 'query' commands in Forge it'd be kinda nice to know if its 'text' or 'not text' and for not text, just return an entity object and let the REST forge-service marshal it as JSON / YAML / XML etc :)